stacks-signer/src/signerdb.rs

```diff
@@ -34,6 +34,73 @@ use stacks_common::util::hash::Sha512Trunc256Sum;
 use stacks_common::{debug, error};
 use wsts::net::NonceRequest;
 
+#[derive(Serialize, Deserialize, Debug, PartialEq, Default)]
+/// Information specific to Signer V1
+pub struct BlockInfoV1 {
+    /// The associated packet nonce request if we have one
+    pub nonce_request: Option<NonceRequest>,
+    /// Whether this block is already being signed over
+    pub signed_over: bool,
```

Comment:
`signed_over` is used in `v0` too: it indicates whether or not the local signer has signed off on the block.